### PR TITLE
UND-52-GlobalHeader

### PR DESCRIPTION
**Pull request by Ashish-srij**
Global Header

### CODE REVIEW

File: stories/assets/js/navigation.js [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+  const megaNavOption = $('.mega-nav-option');
+  const mobLinkWrapper = $('.mob-links');
+  const submenuli = $('.submenu li');
+  const midNav = $('.mid-nav');
+
+  const getSubMenuData = (id) => {
+    const data = subMenus.find((sub) => sub.link.id === id);
+    if (!data.image) {
+      $('.mega-nav-option .mega-sub-description').removeClass('large-6');
+      $('.mega-nav-option .mega-sub-description').addClass('large-10');
```
Comment (surbhi90) [inline stories/assets/js/navigation.js:14]: @Ashish-srij can we write like this $('.mega-nav-option .mega-sub-description').removeClass('large-6').addClass('large-10');

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Organism/Navigation/GlobalHeader/GlobalHeader.scss [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  }
+
+  .mid-nav {
+    display: flex;
+    justify-content: center;
+
+    .menu__wrapper {
+      align-items: center;
+      display: flex;
+      height: 7.1875rem;
```
[the remark this answers is not in the document]
Comment (Ashish-srij) [reply stories/Organism/Navigation/GlobalHeader/GlobalHeader.scss:41]: We need fix height

File: stories/Organism/Navigation/GlobalHeader/GlobalHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import { Logo } from '../../../Atom/Global/Logo/logo';
+import { Languageswitcher } from '../../../Molecules/Navbar/Languageswitcher/Languageswitcher';
+import { Menu } from '../../../Molecules/Navbar/Menu/Menu';
+import './GlobalHeader.scss';
+
+const GlobalHeader = ({
+  text, languageswitcherData, navigationData, rightNavigationData, leftNavigationData, locale,
+}) => {
+  useEffect(() => {
+    console.log(navigationData);
```
[the remark this answers is not in the document]
Comment (surbhi90) [reply stories/Organism/Navigation/GlobalHeader/GlobalHeader.jsx:17]: @Ashish-srij Please remove this console

**Review by surbhi90 — CHANGES_REQUESTED** [review]

**Review by surbhi90 — CHANGES_REQUESTED** [review]

File: stories/assets/scss/_breakpoints.scss [hunk 4]
```diff
@@ -5,7 +5,12 @@
       @content;
     }
   }
-
+  @else if $point==mediumonlytab {
+    /* medium design tab */
+    @media (max-width: 89.9375em) and (min-width:48em){
```
Comment (surbhi90) [inline stories/assets/scss/_breakpoints.scss:10]: @Ashish-srij It should be written like @media (min-width:48em) and (max-width: 89.9375em)

**Review by surbhi90 — CHANGES_REQUESTED** [review]

**Review by surbhi90 — APPROVED** [review]

**Pull request merged**